snuba/cli/cleanup.py

```diff
@@ -55,6 +51,20 @@ def cleanup(
     dataset = get_dataset(dataset_name)
     table = enforce_table_writer(dataset).get_schema().get_local_table_name()
 
-    clickhouse = ClickhousePool(clickhouse_host, clickhouse_port)
-    num_dropped = run_cleanup(clickhouse, database, table, dry_run=dry_run)
-    logger.info("Dropped %s partitions on %s" % (num_dropped, clickhouse_host))
+    if clickhouse_host and clickhouse_port:
+        clickhouse_connections = [ClickhousePool(clickhouse_host, clickhouse_port)]
+    elif not local_dataset_mode():
+        raise click.ClickException("Provide Clickhouse host and port for cleanup")
```

Comment:
Have you verified we never rely on the default in any environment where we may be running in dist mode and changing the settings to connect to the specific node instead of passing the parameters in the CLI ?

Reply:
We do actually pass a ClickHouse host without a port (relying on default) in production right now. I think it's worth keeping this change and making sure we update there before this PR is merged though.